homeassistant/components/fritz/sensor.py

```diff
@@ -34,30 +37,38 @@ def _retrieve_uptime_state(status, last_value):
     return last_value
 
 
-def _retrieve_external_ip_state(status, last_value):
+def _retrieve_external_ip_state(status: FritzStatus, last_value: str) -> Any:
```

Comment:
From `fritzconnection` code it seems to return `str`. You'll probably need to add `# type: ignore[smth]` on the next line.
```suggestion
def _retrieve_external_ip_state(status: FritzStatus, last_value: str) -> str:
```

Reply:
Fixed